src/app/InteractionModelEngine.h

```diff
@@ -335,20 +451,45 @@ class InteractionModelEngine : public Messaging::UnsolicitedMessageHandler,
 
     ObjectPool<CommandHandler, CHIP_IM_MAX_NUM_COMMAND_HANDLER> mCommandHandlerObjs;
     ObjectPool<TimedHandler, CHIP_IM_MAX_NUM_TIMED_HANDLER> mTimedHandlers;
-    ObjectPool<ReadHandler, CHIP_IM_MAX_NUM_READ_HANDLER> mReadHandlers;
     WriteHandler mWriteHandlers[CHIP_IM_MAX_NUM_WRITE_HANDLER];
     reporting::Engine mReportingEngine;
+
+    // CHIP_CONFIG_MAX_FABRICS = actual max number of fabrics + 1 reserved for rotation, so CHIP_CONFIG_MAX_FABRICS - 1 is the
+    // actual maximum number of fabrics supported.
```

Comment:
But during rotation, interactions can happen on the additional fabric, no?

Also, nothing actually enforces the "+1 for rotation" business, so in practice devices can be operational with CHIP_CONFIG_MAX_FABRICS fabrics, as far as I can tell.